native/core/src/parquet/parquet_exec.rs

```diff
@@ -110,6 +110,7 @@ fn get_options(session_timezone: &str) -> (TableParquetOptions, SparkParquetOpti
     let mut spark_parquet_options =
         SparkParquetOptions::new(EvalMode::Legacy, session_timezone, false);
     spark_parquet_options.allow_cast_unsigned_ints = true;
+    spark_parquet_options.case_sensitive = false;
```

Comment:
Not sure if it makes sense to take the value from `spark.sql.caseSensitive` although this is an internal config and false by default

Reply:
I noticed there is a TODO comment above, maybe we can make them configurable in the future https://github.com/apache/datafusion-comet/blob/c58f26162dc8bc6375ac228b88f2da2b1bb686a8/native/core/src/parquet/parquet_exec.rs#L107